airbyte-integrations/connectors/source-google-forms/manifest.yaml

```diff
@@ -0,0 +1,401 @@
+version: 5.15.0
+
+type: DeclarativeSource
+
+description: >-
+  Google Forms is a free online tool from Google that allows users to create
+  custom surveys, quizzes, and forms. It enables easy collection and
+  organization of data by automating responses into a connected Google Sheets
+  spreadsheet. With Google Forms, you can design forms with various question
+  types, share them via email or links, and track responses in real-time, making
+  it ideal for feedback collection, event registration, or educational
+  assessments.
+
+check:
+  type: CheckStream
+  stream_names:
+    - forms
+
+definitions:
+  streams:
+    forms:
+      type: DeclarativeStream
+      name: forms
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: "{{config[\"formId\"]}}"
```

Comment:
Make this a paremeter request:
![image](https://github.com/user-attachments/assets/8329faad-18eb-480f-8518-62e8467d74a6)